editor/package.json

```diff
@@ -11,27 +11,28 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^6.0.3",
-    "@angular/common": "^6.0.3",
-    "@angular/compiler": "^6.0.3",
-    "@angular/core": "^6.0.3",
-    "@angular/forms": "^6.0.3",
-    "@angular/http": "^6.0.3",
-    "@angular/platform-browser": "^6.0.3",
-    "@angular/platform-browser-dynamic": "^6.0.3",
-    "@angular/router": "^6.0.3",
+    "@angular/animations": "^7.1.1",
+    "@angular/cdk": "^7.1.0",
```

Comment:
is this correct that the version numbe differs from All other Angular packages?

Reply:
Yep, it is the latest version currently.